website/docs/r/is_security_group_rule.html.markdown

```diff
@@ -29,7 +29,7 @@ resource "ibm_is_vpc" "example" {
   name = "example-vpc"
 }
 
-resource "ibm_is_security_group" "example" {
+resource "ibm_is_security_group" "example_security_group" {
```

Comment:
we keep just "example" as the name !?

Reply:
This would conflict with the available resource, thus for the config to work named it with resource name addition.

Reply:
done.